lib/apiservers/engine/backends/container.go

```diff
@@ -478,50 +503,35 @@ func (c *Container) containerStart(name string, hostConfig *container.HostConfig
 	}
 
 	// get a handle to the container
-	var getRes *containers.GetOK
-	getRes, err = client.Containers.Get(containers.NewGetParamsWithContext(ctx).WithID(name))
+	handle, err := c.Handle(name, paramName)
```

Comment:
The model for the handles is that there is one per operation and the handle is only valid for that one operation.
This allows separation of the port layer components in the future without major changes to the callers.